src/rotating_gate.c

```diff
@@ -740,7 +740,7 @@ static u8 RotatingGate_CreateGate(u8 gateId, s16 deltaX, s16 deltaY)
 
     template.tileTag = gate->shape + ROTATING_GATE_TILE_TAG;
 
-    spriteId = CreateSprite(&template, 0, 0, 0x93);
+    spriteId = CreateSprite(&template, 0, 0, 0x93); // 0x93 is above shadows (0x94)
```

Comment:
Do we need this comment?

Reply:
I'd be tempted to write this as `0x94 - 1` (similar to the `0x94 + 1` below)... Except I agree with your comment elsewhere about magic numbers. If we had `#define OW_SHADOW_SUBPRIORITY 0x94` or whatever, then it's `OW_SHADOW_SUBPRIORITY - 1` which maybe doesn't need the comment? Lower (sub)priority = above is consistent across the codebase.